### PR TITLE
Updated CF stack diagram and setup document

### PR DESCRIPTION
**Pull request by vnandwana**
CF product now uses a single AMI for both KB and Kaui -- Updated CF stack diagram and setup document.

### CODE REVIEW

File: userguide/aws/how-to-set-up-a-cloud-formation-system.adoc [hunk 1]
```diff
@@ -5,11 +5,11 @@
 
 This document describes the procedures for setting up Kill Bill under AWS using the CloudFormation option. This is one of two recommended alternatives for production use. The multi-tier option requires more setup than CloudFormation, but provides more control over the deployment. The procedures in this document are based on the options recommended by Kill Bill.
 
-Deployment via CloudFormation leverages the capabilites of the AWS infrastructure to provide a robust *production ready* deployment with a single click. The entire CloudFormation configuration, or stack, is defined by a *CloudFormation Template*.
+Deployment via CloudFormation leverages the capabilities of the AWS infrastructure to provide a robust *production ready* deployment with a single click. The entire CloudFormation configuration, or stack, is defined by a *CloudFormation Template*.
 
 The features of the CloudFormation system include:
 
-* Both Kill Bill and Kaui instances can be scaled up or down using AWS autoscaling groups
+* Kill Bill and Kaui instance can be scaled up or down using AWS autoscaling groups
```
Comment (reshmabidikar) [inline userguide/aws/how-to-set-up-a-cloud-formation-system.adoc]: Consider rephrasing as 
```suggestion
* The instance having Kill Bill/Kaui installed can be scaled up or down using AWS autoscaling groups
```
or something similar to make it clear that KB/Kaui are on the same instance.

File: userguide/aws/how-to-set-up-a-cloud-formation-system.adoc [hunk 2]
```diff
@@ -29,7 +29,7 @@ image::../assets/aws/cf_stack.png[align=center]
 
 All resources for this system run within a single AWS *Virtual Private Cloud (VPC)*, providing a dedicated block of IP addresses which must be located in a single *region*. The cloud is partitioned into *availability zones*, which are accessed by *subnets*. The resources must be distributed over at least two availability zones.
 
-Multiple instances of Kaui and the Kill Bill server are deployed in the VPC, each running on its own Ubuntu Linux server. AWS autoscaling is used to increase and decrease the number of instances for each package as needed.
+A single instance of Kill Bill and Kaui servers is deployed in the VPC, running on an Ubuntu Linux server. AWS autoscaling is utilized to dynamically adjust the number of instances for each package based on demand, allowing for efficient scaling.
```
Comment (reshmabidikar) [inline userguide/aws/how-to-set-up-a-cloud-formation-system.adoc]: Same suggestion as above. Consider rephrasing to:
```suggestion
A single instance having the Kill Bill/Kaui server is deployed in the VPC, running on an Ubuntu Linux server. AWS autoscaling is utilized to dynamically adjust the number of instances for each package based on demand, allowing for efficient scaling.
```

**Review by reshmabidikar — APPROVED** [review]
Looks good, I've left some minor review comments. You can make changes if those comments make sense, otherwise good to be merged as it is.

**Comment by vnandwana** [comment]
@reshmabidikar Thank you for the feedback. I have updated the doc, kindly take a look when you get a chance.

**Review by sbrossie — APPROVED** [review]

**Pull request merged**